src/static/chemiscope.css

```diff
@@ -233,8 +233,11 @@
 
 .chsp-properties-table {
     height: 100%;
-    width: 80%;
+    width: 90%;
     margin: auto;
+    overflow-x: auto;
```

Comment:
```suggestion
```

from my tests, I noticed that opening the info panel started to lag a bit, after this line was added (it is not instant as currently in main). since we now limit numeric values to 4 characters after the point, overflow is no longer really an issue. so I propose removing this line